addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -517,6 +517,8 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
 
 field_registry.add('html', FieldHtml);
 
+// allow to use widget = 'text' on html field
+field_registry.map.text.prototype.supportedFieldTypes.push('html');
```

Comment:
I think you can just modify the line `supportedFieldTypes: ['text'],` into `supportedFieldTypes: ['text', 'html'],` in web, no? The feature does not seem related to web_editor.

Reply:
don't like the idea to modify web to add support of feature in web_editor.

If you just have web, you are not aware of this htmlfield.

Seems more correct like that.

Reply:
after the real question, is... Why fieldHtml is not in web instead of web_editor ;)

Reply:
You don't use web_editor here, I think you are confused with the html field *widget* :wink: Your PR is making the text field widget of *web* available for python html field (of *base*). No web_editor involved to me. Or I misunderstand the original problem.

Reply:
'html' in your new line does not refer to the web_editor 'html' field but to the python 'html' field type